Source/MQTTnet.Extensions.WebSocket4Net/WebSocket4NetMqttChannel.cs

```diff
@@ -85,7 +85,12 @@ public async Task ConnectAsync(CancellationToken cancellationToken)
             {
                 foreach (var certificate in _webSocketOptions.TlsOptions.Certificates)
                 {
+#if WINDOWS_UWP
                     certificates.Add(new X509Certificate(certificate));
```

Comment:
I am not sure but this looks wrong to me. In all other places you are using byte[] for UWP but here you use X509Certificate directly.

Reply:
for UWP `certificate` variable it's byte array, but in this particular file was no platform implementation for uwp, it using `X509Certificate` for all platforms